index.html

```diff
@@ -49,7 +49,7 @@
   <!-- Navigation -->
   <nav class="navbar navbar-expand-lg bg-secondary fixed-top text-uppercase" id="mainNav">
     <div class="container">
-      <a class="navbar-brand js-scroll-trigger" href="#page-top"><img src="img/logo.png" /></a>
+      <a class="navbar-brand js-scroll-trigger" href="#page-top"><img src="img/sponsors/logo.png" /></a>
```

Comment:
This seems accidental?

Reply:
Oopsie poopsie, yes - Wrong find+replace 👼

Reply:
Fixed:
![image](https://user-images.githubusercontent.com/4345663/61872072-b0966900-aee2-11e9-9812-d383c9bea856.png)